### PR TITLE
Resolves occasions list formatting is off due to missing data

### LINKED ISSUE #11760: Table Data items shift to the left
### Please confirm the following

- [X] I agree to follow this project's [code of conduct](https://docs.ansible.com/ansible/latest/community/code_of_conduct.html).
- [X] I have checked the [current issues](https://github.com/ansible/awx/issues) for duplicates.
- [X] I understand that AWX is open source software provided for free and that I might not receive a timely response.

### Summary

They shift to the left if there is missing data

### AWX version

19.5

### Select the relevant components

- [X] UI
- [ ] API
- [ ] Docs

### Installation method

docker for mac

### Modifications

no

### Ansible version

_No response_

### Operating system

_No response_

### Web browser

_No response_

### Steps to reproduce

Create users with no first name, or last name. Go to users list.

### Expected results

Table columns render properly
Screen Shot 2022-02-16 at 2 01 08 PM

### Actual results

Screen Shot 2022-02-16 at 1 56 51 PM

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by AlexSCorey**
##### SUMMARY
Resolves #11760 
##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
 - UI

##### AWX VERSION
Screen Shot 2022-02-17 at 2 00 30 PM

### CODE REVIEW

File: awx/ui/src/screens/User/UserList/UserList.test.js [hunk 1]
```diff
@@ -124,6 +159,15 @@ describe('UsersList with full permissions', () => {
     expect(wrapper.find('ToolbarAddButton').length).toBe(1);
   });
 
+  test('Last user should have no first name or last name and the row items should render properly', async () => {
+    await waitForElement(wrapper, 'ContentLoading', (el) => el.length === 0);
```
Comment (nixocio) [inline awx/ui/src/screens/User/UserList/UserList.test.js:163]:
```suggestion
  
```

File: awx/ui/src/screens/User/UserList/UserList.test.js [hunk 2]
```diff
@@ -124,6 +159,15 @@ describe('UsersList with full permissions', () => {
     expect(wrapper.find('ToolbarAddButton').length).toBe(1);
   });
 
+  test('Last user should have no first name or last name and the row items should render properly', async () => {
```
Comment (nixocio) [inline awx/ui/src/screens/User/UserList/UserList.test.js:162]:
```suggestion
  test('Last user should have no first name or last name and the row items should render properly', () => {
```

File: awx/ui/src/screens/User/UserList/UserList.test.js (hunk 1 above)
Comment (nixocio) [reply awx/ui/src/screens/User/UserList/UserList.test.js:163]: beforeEach is already waiting for element.

**Review by nixocio — APPROVED** [review]
Good job @AlexSCorey, minor suggestion.

**Review by InnocentK — APPROVED** [review]
Looks good

**Pull request merged**